### PR TITLE
updated latest changes from wrapper

### PR DESCRIPTION
**Pull request by awaseem**

### CODE REVIEW

File: Example/android/settings.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 rootProject.name = 'Example'
 include ':react-native-charts-wrapper'
-project(':react-native-charts-wrapper').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-charts-wrapper/android')
```
Comment (kirandwarkan) [inline Example/android/settings.gradle]: Might want to revert this one.

File: Example/app/PieChartScreen.js [hunk 2]
```diff
@@ -55,15 +58,15 @@ class PieChartScreen extends React.Component {
...
   }
 
   render() {
     return (
-
-      <View style={{flex: 1}}>
-
-        <View style={{height:80}}>
-          <Text> selected entry</Text>
+      <SafeAreaView style={{flex: 1}}>
```
Comment (kirandwarkan) [inline Example/app/PieChartScreen.js]: Need to check that this still works for us on Android,.

File: Example/android/settings.gradle (hunk 1 above)
Comment (awaseem) [reply Example/android/settings.gradle]: oh, good catch!!

File: Example/app/AxisLineChartScreen.js [hunk 3]
```diff
@@ -98,6 +98,8 @@ class AxisLineChartScreen extends React.Component {
     } else {
       this.setState({...this.state, selectedEntry: JSON.stringify(entry)})
     }
+
+    console.log(event.nativeEvent)
```
Comment (rziehl) [inline Example/app/AxisLineChartScreen.js]: Was this just for debugging?

File: Example/app/BarChartScreen.js [hunk 4]
```diff
@@ -54,6 +55,8 @@ class BarChartScreen extends React.Component {
     } else {
       this.setState({...this.state, selectedEntry: JSON.stringify(entry)})
     }
+
+    console.log(event.nativeEvent)
```
Comment (rziehl) [inline Example/app/BarChartScreen.js]: Like my other comment, was this also for debugging?

File: Example/app/BubbleChartScreen.js [hunk 5]
```diff
@@ -79,6 +79,8 @@ class BubbleChartScreen extends React.Component {
     } else {
       this.setState({...this.state, selectedEntry: JSON.stringify(entry)})
     }
+
+    console.log(event.nativeEvent)
```
Comment (rziehl) [inline Example/app/BubbleChartScreen.js]: Debugging?

**Pull request merged**